example/build.yaml

```diff
@@ -0,0 +1,30 @@
+# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
+builders:
+
+  copyBuilder:
+    target: ":example"
```

Comment:
Should we add comments for these now? Also I think you can actually omit `target`, its only needed for bazel and is just confusing right now imo.

Reply:
https://github.com/dart-lang/build/tree/master/build_config Not commented as required/optional here...

Reply:
See https://github.com/dart-lang/build/issues/1258

Reply:
I meant all the keys - to document what each one means